packages/react/src/components/NumberInput/NumberInput-test.js

```diff
@@ -466,13 +470,20 @@ describe('NumberInput', () => {
         downArrow.simulate('click');
         expect(onClick).toBeCalled();
         expect(onChange).toBeCalled();
-        expect(onClick).toHaveBeenCalledWith(expect.anything(), 'down');
+        expect(onClick).toHaveBeenCalledWith(
+          expect.anything(),
+          'down',
+          expect.anything()
```

Comment:
since this is the value should we make this assertion more specific? (number and/or string)

Reply:
I think, ideally, yes @emyarod I'm not entirely sure why `direction` isn't passed down as a param here originally. I can add it if we want them to be the same. thoughts? @joshblack

Reply:
was this meant to be a reply to a different review comment (https://github.com/carbon-design-system/carbon/pull/8243/files/18dba3ae39f5e3e6826c9b48536849a7c9dc47c7#r605683234)?

Reply:
oops, yes. 😓 @emyarod